pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest2.java

```diff
@@ -1521,6 +1522,17 @@ public void testMaxTopicsPerNamespace() throws Exception {
         }
 
         // check producer/consumer auto create partitioned topic
+        final Function<Producer<byte[]>, Producer<byte[]>> send = (p) -> {
```

Comment:
Why are you modifying an existing test? 
It is okay to refactor and reuse an existing test but in this case probably you are altering the behaviour of the existing guest and also you are not sure that you are testing your feature properly

Reply:
> Why are you modifying an existing test?

Because this feature is modifying existing behavior. Currently, a partitioned producer connects to all partitions. Some cases of tests suppose "partitioned producer connects to all partitions(and also create all partition automatically)".
In this change, I modified this procedure to lazy-loading and modified test cases to follow it([here](https://github.com/apache/pulsar/pull/10279/files#diff-5277af06358fe0abda3ede298643c934416cbfec27ab0e77d4c924320cd08e8b) is a part of test codes for this feature).

If we should not change existing behavior, I'll change this feature to be able to stop.